misc/roundtrip

```diff
@@ -53,7 +53,7 @@ expandEscapeSequences()
 :again
     s/\\/__BACKSLASH__/
     T out
-    s/__BACKSLASH__\\/__LASTBACKSLASH__/;	t again
+    s/__BACKSLASH__\\/__LITBACKSLASH__/;	t again
```

Comment:
Could you tell me why this change is needed?

Reply:
It is not needed per se, but I changed the name for clarity as it's not only used for the last backslash but for all literal backslashes